samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -151,6 +276,17 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 		const size_t bis_count = base->subgroups[i].bis_count;
 
 		printk("Subgroup[%zu] has %zu streams\n", i, bis_count);
+
+#if defined(CONFIG_LIBLC3)
+		int8_t ret;
+		struct bt_audio_codec_cfg *codec_cfg = &base->subgroups[i].codec_cfg;
+
+		ret = lc3_enable(codec_cfg);
```

Comment:
We should check that `codec_cfg->id == BT_HCI_CODING_FORMAT_LC3` before enabling the LC3 decoder